include/openenclave/bits/report.h

```diff
@@ -103,6 +103,12 @@ typedef struct _oe_identity
     /** The Product ID for the enclave.
      * For SGX enclaves, this is the ISVPRODID value. */
     uint8_t product_id[OE_PRODUCT_ID_SIZE];
+
+    /** The unique ID for the device. */
+    uint8_t device_unique_id[OE_UNIQUE_ID_SIZE];
```

Comment:
This is only set for ARM, correct? Consider adding a comment explaining what this ID is, how it differs from the others, and what its semantics are for SGX and TZ.

Reply:
Not ARM but CyRes. Will extend, but wanted to start the discussion if OE folks will like this or not.